sgx-tools/Makefile

```diff
@@ -12,9 +12,13 @@ COMMIT ?= $(if $(shell git status --porcelain --untracked-files=no),"${COMMIT_NO
 
 VERSION := ${shell cat ./VERSION}
 SOURCES := $(shell find . 2>&1 | grep -E '.*\.(c|h|go)$$')
-$(APP): $(SOURCES)
+PROTO_DIR := proto
+$(APP): $(SOURCES) $(patsubst %.proto,%.pb.go,$(wildcard $(PROTO_DIR)/*.proto))
 	$(GO) build -buildmode=pie $(EXTRA_FLAGS) -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION} $(EXTRA_LDFLAGS)" -o $(APP) .
 
+%.pb.go: %.proto
+	@protoc --proto_path=$(PROTO_DIR) --go_out=$(PROTO_DIR) $^
```

Comment:
If this function is so simple and thus don't worthy merging to code base, it is fine to place new protobuf message inside sgx-tools. However, this single function is complete enough, so please move the new protobuf messages to intelsgx package. Yes, I don't this this is pr is RFC.